### PR TITLE
changed flyby tgui

### PR DESCRIPTION
**Pull request by mullenpaul**
# About the pull request

Slight change to the TGUI for dropship flight controls. I've had feedback that it is a bit unintuitive. This change should resolve novice POs accidentally launching flyby in a panic.

# Explain why it's good for the game
Better UIs make everyone happier.
# Testing Photographs and Procedure
<details>
<summary>Screenshots & Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

image

</details>

# Changelog
:cl:
ui: tweaked flyby controls to make them in line with other destinations
/:cl:

### CODE REVIEW

File: code/modules/shuttle/computers/dropship_computer.dm [hunk 1]
```diff
@@ -359,23 +368,24 @@
...
-				to_chat(user, SPAN_WARNING("You don't have the skill to perform a flyby."))
-				return FALSE
 			var/is_optimised = FALSE
 			// automatically apply optimisation if user is a pilot
 			if(skillcheck(user, SKILL_PILOT, SKILL_PILOT_EXPERT))
 				is_optimised = TRUE
 			update_equipment(is_optimised)
-			if(is_set_flyby)
+			var/dockId = params["target"]
+			if(dockId == DROPSHIP_FLYBY_ID)
```
Comment (harryob) [inline code/modules/shuttle/computers/dropship_computer.dm:377]:
```suggestion
			var/dock_id = params["target"]
			if(dock_id == DROPSHIP_FLYBY_ID)
```

**Review by harryob — CHANGES_REQUESTED** [review]

**Review by fira — APPROVED** [review]

**Comment by IsProbablyCatto** [comment]
Isn't tested. Flyby is now coded as a destination, which lasts 30 seconds instead of 200 seconds.

**Comment by cuberound** [comment]
jeh none tested it and you just merged it... indeed does not work as intended

**Comment by mullenpaul** [comment]
> Isn't tested. Flyby is now coded as a destination, which lasts 30 seconds instead of 200 seconds.

This was tested, it introduced a regression.

Flybys we're always coded as destinations. Behind the scenes the shuttle is forced to transit to it's existing landing zone. This work was too change how the UI tells the backend how to do this.

**Comment by dadlips** [comment]
There is no longer a cancel flyby button, is this intented? Our only option is to wait out the full 200+ seconds, and with a medevac'd marine this can be bad. 
image

**Pull request merged**